src/cmd-kola

```diff
@@ -12,18 +12,21 @@ fi
 
 # XXX: teach to kola to auto-detect based on prefix; see discussions in
 # https://github.com/coreos/coreos-assembler/pull/85
-distro=
+args=
 bn=$(basename "${latest_qcow}")
 if [[ ${bn} = fedora-coreos-* ]]; then
-    distro="-b fcos"
+    args="-b fcos"
 elif [[ ${bn} = rhcos-* ]]; then
-    distro="-b rhcos --ignition-version v2"
+    args="-b rhcos"
 else
-    echo "WARNING: Failed to detect distro, use -b to specify"
+    echo "WARNING: Failed to detect args, use -b to specify"
+fi
+if [ $(disk_ignition_version "${latest_qcow}") = "2.2.0" ]; then
```

Comment:
Notably with this change `--ignition-version v2` will only be specified in the basename matches `rhcos-410.8*` instead of just `rhcos-*`. Is that intentional?

Reply:
Yes